examples/tutorial/x/blog/handlers_test.go

```diff
@@ -0,0 +1,14 @@
+package blog
+
+import (
+	"testing"
+)
+
+func TestCreateBlogMsgHandlerCheck(t *testing.T) {
```

Comment:
I would concentrate on testing Deliver. 
1-2 test cases that Check works on a good case and fails on a bad should be fine there.

validate is covered in those other cases, but if there is complex logic, yes, maybe better to table test it in isolation.